src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/CaseDataDetails.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.digital.ho.hocs.casework.domain.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import uk.gov.digital.ho.hocs.casework.domain.repository.JsonConfigFolderReader.CaseTypeObject;
+
+import java.util.List;
+import java.util.Map;
+
+public class CaseDataDetails implements CaseTypeObject<Map<String, List<CaseDataDetails.Fields>>> {
+
+    private final String type;
+    private final Map<String, List<Fields>> groups;
+
+    @JsonCreator
+    public CaseDataDetails(@JsonProperty("type") String type,
+                           @JsonProperty("groups") Map<String, List<Fields>> fields) {
+        this.type = type;
+        this.groups = fields;
+    }
+
+    @Override
+    public String getType() {
+        return type;
+    }
+
+    @Override
+    public Map<String, List<Fields>> getValue() {
+        return groups;
+    }
+
+    @Getter
+    public static class Fields {
+
+        private final String name;
+        private final String label;
+        private final String component;
+        private final Map<String, Object> props;
```

Comment:
The reason why props is here and not a slimmed down version is because we should support the ability for this to show the fields outside of the mapped-display component. Having props here (which we can cut down some of the values) - acts as a natural grouping.

Reply:
I've left all of the props for now incase we want to do something fancy - we can always revisit and remove unnecessary ones.